ert_storage/endpoints/responses.py

```diff
@@ -0,0 +1,41 @@
+from uuid import uuid4, UUID
+import pandas as pd
+from fastapi import (
+    APIRouter,
+    Depends,
+)
+from fastapi.responses import Response
+from pandas.core.frame import DataFrame
+from ert_storage.database import Session, get_db, HAS_AZURE_BLOB_STORAGE
+from ert_storage import database_schema as ds
+
+router = APIRouter(tags=["response"])
+
+
+@router.get("/ensembles/{ensemble_id}/responses/{response_name}/data")
+async def get_ensemble_response_dataframe(
+    *, db: Session = Depends(get_db), ensemble_id: UUID, response_name: str
+) -> Response:
+    ensemble = db.query(ds.Ensemble).filter_by(id=ensemble_id).one()
+    records = (
+        db.query(ds.Record)
+        .filter_by(
+            ensemble_pk=ensemble.pk,
+            name=response_name,
+            record_class=ds.RecordClass.response,
+        )
+        .filter(ds.Record.realization_index != None)
+    ).all()
+    df_list = []
+    for record in records:
+        data_df = pd.DataFrame(record.f64_matrix.content)
```

Comment:
What happens if one if the f64_matrixes are multidimensional?

Reply:
It will be always `1D` as far as I understood (eg. via flattening) if not then  everything falls apart since observations would need to support `nD` data too.

Reply:
As we talked about, I am not sure how that assumption will last in to the future. We might set us up to fail if we assume everything fits into DataFrames.

Reply:
Yes, but if we need to have a more general definition we might change it when the time comes. Currently this will make sure that we got a exception: `data_df.index = [record.realization_index]`